ingestion-edge/ingestion_edge/publish.py

```diff
@@ -135,6 +173,10 @@
         ).items()
         if value is not None
     }
+    # assert valid pubsub message
+    for value in attrs.values():
+        if len(value.encode("utf8")) > 1024:
```

Comment:
Can you add a comment here linking to PubSub documentation that specifies this limit?